ci/src/update-helm-chart-deps.mjs

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env zx
+
+import { config } from 'dotenv'
+import envalid, { bool, json, str } from 'envalid'
+import fs from 'fs/promises'
+import yaml from 'js-yaml'
+import semver from 'semver'
+import { $ } from 'zx'
+
+async function main() {
+  config()
+  const env = envalid.cleanEnv(process.env, {
+    CI_UPDATE_TYPE: str({
+      desc: 'Path to the YAML file to validate',
+      choices: ['patch', 'minor', 'major'],
+      default: 'minor',
+    }),
+    CI_HELM_CHART_NAME_FILTER: json({ desc: 'A list of names in json format', default: [] }),
+    CI_GH_CREATE_PR: bool({ desc: 'Create Github PR', default: true }),
+    CI_GIT_BASELINE_BRANCH: str({ desc: 'A baseline git branch', default: 'main' }),
+    CI_GIT_LOCAL_BRANCH_ONLY: bool({ desc: 'Perform changes only on local branches', default: false }),
+  })
+
+  // Path to the Chart.yaml file
+  const chartFile = '../chart/chart-index/Chart.yaml'
+  const chartsDir = '../charts'
```

Comment:
Paths should start with `../../<path>` instead of `../<path>`.

Reply:
Are you sure?

The script is design to be run from the `ci` directory. I run it also with GitHub actions and it worked

Reply:
You're right I run it in the `ci/src` 👍🏻